<ui version="4.0" >
 <class>SkipModeWidget</class>
 <widget class="QWidget" name="SkipModeWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Skip Mode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="counterLabel" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>counter</string>
     </property>
     <property name="buddy" >
      <cstring>counterComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="counterComboBox" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="precountLabel" >
     <property name="text" >
      <string>precount</string>
     </property>
     <property name="buddy" >
      <cstring>precountSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDoubleSpinBox" name="precountSpinBox" >
     <property name="minimum" >
      <double>0.100000000000000</double>
     </property>
     <property name="maximum" >
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="thresholdLabel" >
     <property name="text" >
      <string>threshold</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="thresholdSpinBox" >
     <property name="maximum" >
      <double>10000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
